extensions/wikia/AdEngine/js/template/bfaaDesktop.js

```diff
@@ -113,6 +145,41 @@ define('ext.wikia.adEngine.template.bfaaDesktop', [
 		}
 
 		log(['show', params.uap], log.levels.info, logGroup);
+
+		// TODO
+		var adHeight = 406;
+
+
+		$('#TOP_LEADERBOARD').css({
+			height: adHeight + 'px',
+			padding: 0,
+			position: 'fixed',
+			top: 0,
+			transition: 'none',
+			width: '100%'
+		});
+
+		$('#DirectGpt_TOP_LEADERBOARD').css({
+			position: 'static'
+		});
+
+		$('#globalNavigation').css({
+			top: adHeight + 'px',
+			position: 'fixed'
+		});
+
+		$('#WikiaTopAds').css({
+			height: adHeight + 'px',
+			'z-index': 6
+		});
+
+		win.addEventListener('scroll', function() {
+			bfaaMagic(params);
+		});
+		// TODO onready
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)